src/osm_configurator/model/project/configuration/attribute_enum.py

```diff
@@ -69,6 +69,12 @@ def calculate_attribute_value(self, category: Category, tags: Dict[str, str]):
         """
         return self.value[2](category, tags)
 
+    def equals(self, name: str) -> Attribute | None:
```

Comment:
better name or descirbe in pydoc what this method does.
A "normal" equals method would return a boolean.